docs/related/amf-conversion.mdx

```diff
@@ -1,9 +1,193 @@
 ---
 id: amf_conversion
 title: AMF Conversion
-hide_title: true
 ---
 
-# AMF Conversion
+:::caution
+The behaviour expressed in this documentation is currently a work in progress.
+If any bugs are found that do not satisfy these requirements,
+please file an issue in the [aml-org/amf repository](https://github.com/aml-org/amf/pull/715).
+:::
 
+# What is WebAPIs conversion?
 
+Conversion stands for the capability of parsing an API from one specification vendor into an API graph and then rendering said graph as a different vendor’s specification.
+
+Different specifications can have different required fields, and the same information conceptually may not be described in the same way.
+This is the reason why in a *cross parsing-rendering process*, the AMF graph needs to go through a transformation called *“Compatibility resolution”*.
```

Comment:
I'm fairly certain that conversion is the same thing as "cross parsing-rendering process". However, the way this is constructed it seems like "cross parsing-rendering process" is introduced here without definition.

Maybe:
Conversion (*cross parsing-rendering process*) stands for the capability of parsing an API from one specification vendor into an API graph and then rendering said graph as a different vendor’s specification.